src/commands/kat-tests-run.yaml

```diff
@@ -0,0 +1,25 @@
+parameters:
+    chart:
+        type: string
+steps:
+  - run:
+      name: "initialize helm"
+      command: |
+        /tmp/bin/helm init -c
+  - run:
+      name: "start kube-app-testing tool"
+      command: |
+        PATH="/tmp/bin:$PATH" /tmp/kube-app-testing.sh -c << parameters.chart >>
+  - run:
+      name: ""
+      command: |
+        if [[ -d /tmp/kat ]]; then sudo chown -R circleci.circleci /tmp/kat; fi
+  - save_cache:
+        key: v2-python-cache-{{ checksum "test/kat/Pipfile.lock" }}
```

Comment:
Why is it necessary to save the cache here?

Reply:
Test executor might fetch additional pip packages during run, so you have to cache them after the run.

Reply:
I don't see why we need to cache them at all - it seems like premature optimization to me to cache something when we are not sure if it actually improves build times.

Reply:
I checked that before implementig caching, gives nice speed boost.